### ISSUE TITLE
[Feature]: Device media playback in Android Auto

### ISSUE THREAD

**Issue by BabyBenefactor** [fast4x/RiMusic#628]
### Description

In Rimusic, there is the ability to play songs not only from Youtube Music but also from the local device's storage. In Android Auto there is the ability to play offline songs, but not those from the device itself. 
Since not alot of device playback apps (like Auxio) support Android Auto this could be a very nice enhancement for the all-in-one music playing app.

### Suggested Solution

Add another option in the 'Playlists' menu called 'On device'.

### Additional Context

I don't know if this is possible to implement, nor do I posess the technical knowledge to implement it myself. I am willing to help test this feature out. 
Thanks for the amazing app btw, beats Innertune by a mile.

**Comment by fast4x** [comment]
Hi, On device is just in Lybrary...

**Comment by fast4x** [comment]
You’re right, excuse me, I better check the configuration for Android Auto

**Comment by fast4x** [comment]
Even if you are not interested, the ticket can remain open.

**Comment by fast4x** [comment]
Soon I'll add artists..
